### ISSUE TITLE
Trying to run 2_fullyconnected.ipynb but neural net doesn't learn

### ISSUE THREAD

**Issue by Vo1ishe** [tensorflow/tensorflow#13362]
I'm trying to run that notebook as it is but model doesn't learn at all. Initially I got right results like in sample but now I'm instantly getting such output when train a multinomial logistic regression using simple gradient descent:

Initialized
Loss at step 0: 24.628622
Training accuracy: 9.7%
Validation accuracy: 10.0%
Loss at step 100: 35.364712
Training accuracy: 9.9%
Validation accuracy: 9.6%
Loss at step 200: 38.703053
Training accuracy: 9.9%
Validation accuracy: 10.0%
Loss at step 300: 30.087294
Training accuracy: 10.1%
Validation accuracy: 10.0%
Loss at step 400: 35.924911
Training accuracy: 9.9%
Validation accuracy: 10.0%
Loss at step 500: 38.568333
Training accuracy: 10.5%
Validation accuracy: 10.0%
Loss at step 600: 31.410255
Training accuracy: 10.8%
Validation accuracy: 10.0%
Loss at step 700: 42.134827
Training accuracy: 10.1%
Validation accuracy: 10.0%
Loss at step 800: 26.819206
Training accuracy: 9.9%
Validation accuracy: 10.0%
Test accuracy: 10.0%

When I switch to switch to stochastic gradient descent training I get simmilar output. Restarting kernel and even VM doesn't work. I also tried to delete preveously generated pickle and generate it from scratch but it also doesn't help.

What might cause my neural net to stop learning at some point?

**Comment by Noahyt** [comment]
The network is guessing at random.  This is MNIST so 10 possible choices -> 10% of getting anything right completely at random.  
Some basic troubleshooting:
Look at output. is it predicting a constant value?  
Come up with fake data, see what network does.

**Comment by arinbjornk** [comment]
I think you will get more help over at [Stack Overflow](
https://stackoverflow.com/questions/tagged/tensorflow) since this is probably not a bug
